docs/software/cloud.md

```diff
@@ -230,6 +230,33 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
 
     Your user data directory will stay after deinstallation. As well a database backup will be saved to your user data directory. Thus you can easily restore your instance by reinstalling Nextcloud and restore the database dump.
 
+    **How can I check my OPcache status**
+
+    Nextcloud has a speedup option via the **PHP OPcache** feature. If it is not setup in an appropriate manner, the Nextcloud settings overview dialog show *"PHP OPcache not properly configured"* in combination with needed settings. In the internet many further hints are available to optimize the `php.ini`. DietPi installs appropriate `ini` files by default which supersede the standard `php.ini` file (`97-dietpi.ini` and `98-dietpi-nextcloud.ini`).  
+    Often there is the question which of the various `php.ini` files on the system is the valid one. DietPi therefore has helping files in its `/var/www` directory to give some information about this:  
```

Comment:
The files in `/var/www` are helpful to check the effective PHP configuration, but not to know which ini file to use. It simply lists all of them while one of them should be edited 😉.

Generally:
- The OPcache warnings from the Nextcloud admin panel are nonsense. Those match the PHP defaults anyway, but are much higher than required for Nextcloud while possibly too low when running other PHP applications. So it is wrong to recommend users doing anything about them, until they really show reasonable warnings based on actual OPcache usage: https://github.com/nextcloud/server/pull/27403
- Even that the warnings are wrong, since users would complain seeing them, `dietpi-software` configures it to mute the related Nextcloud warnings. If this is not the case, then instead of doing a manual fix, it should be reported to us to have it fixed our end.

However, there are cases where changing the PHP config is reasonable, especially when multiple PHP applications are used, but IMHO there is only one way to do it properly: Creating an own "module" with priority 99 and enabling it. This however should be placed in a tab at PHP, not here at Nextcloud. Same with the info about `opcache.php`, `apc.php` and `phpinfo.php` since those are not related to any specific PHP application. We may however link our PHP docs from PHP application docs, similar like we discussed for MariaDB.

Reply:
@MichaIng: In this case the setting of opcache.memory_consumption ist not set properly.
In the file `97-dietpi.ini` it is set to 33 MB, instead of using `opcache.memory_consumption=128`.

I added an own file 99-myNextcloudSettings.ini with this line, then the warnings are gone.
Next steps?